.github/workflows/generate-zip.yml

```
@@ -0,0 +1,40 @@
+name: Generate ZIP file
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+    generate-zip-file:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Checkout
+              uses: actions/checkout@v3
```

We'll want to ensure we're using the latest versions of each action

```suggestion
              uses: actions/checkout@v4
```